gtfs/gtfs_utils/gtfs_utils/configuration.py

```diff
@@ -70,7 +70,7 @@ class Configuration:
     files: FilesConfiguration = None
     s3: S3Configuration = None
     use_data_from_today: bool = True
-    date_range: List[str] = None
+    date_range: Iterable[str] = ('',)
```

Comment:
Why did you change to iterable?

Reply:
For it to not be a required field (if some one wants just "use_today_data"), it must have a default value.
Since it is impossible to set a mutable as default values in `dataclass` (only with a default factory) I used tuple.
because there is a type check in `dict_to_dataclass` the default value must be an instance of the hinted type.

Reply:
And what about using `dataclasses.field(default_factory=list)` as a default value?
It just doesn't feel right to change all this only for making it non-required. Doesn't JSON schema have an option to set an optional field?

Reply:
Ok I found why I had a problem with the default_factory, will commit soon.
BTW, I noticed [this commit](https://github.com/hasadna/open-bus/commit/821e3b5d111951a84a3e1bf9715f55eda9864c3b) where you allowed configuration keys that are not in the data class. right now the JSON Schema does not allow that. I think that if someone adds a new key it should be added to the schema and docs as well.

Reply:
Agreed.